File: program2.m
x=[]
lx=input("Enter x array length:")
h=0
for i=1:lx
  l=input("Enter lx:");
  x=[x,l];
 endfor
 ly=input("enter ly");
 t=0:0.01:ly
 for i=1:lx
   h+=x(i)*cos(2*pi*t*i)
 endfor
plot(h)

File: program2.py
import numpy as np
import matplotlib.pyplot as plt

x = []
lx = int(input("Enter x array length: "))
for i in range(lx):
    l = float(input("Enter lx: "))
    x.append(l)

ly = float(input("Enter ly: "))
t = np.arange(0, ly, 0.01)
h = np.zeros(len(t))
for i in range(lx):
    h += x[i] * np.cos(2 * np.pi * t * (i + 1))

plt.plot(t, h)
plt.show()
